content/docs/15.how-to-guides/long-running-intensive-tasks.md

```diff
@@ -0,0 +1,162 @@
+
+In this guide, we will focus on executing long-running tasks on **Kubernetes** using Kestra.
+
+Kubernetes pods are a great fit due to the control and flexibility they provide. With Kubernetes, you can precisely define resource requirements, permissions, namespace, handle workload identity, and ensure proper networking for your tasks. Pods can also easily access other Kubernetes services hosted on the cluster such as databases, storage and applications.
+
+As an exmample, we will use a [dbt job](https://docs.getdbt.com/docs/running-a-dbt-project/run-your-dbt-projects) to demonstrate how Kestra can help you execute complex tasks on Kubernetes with resource requirements.
+
+## Schedule task in a Kubernetes pod using podCreate
+
+Kestra's [podCreate](https://kestra.io/plugins/plugin-kubernetes/tasks/io.kestra.plugin.kubernetes.podcreate) task allows you to launch a Kubernetes pod directly by providing the complete Kubernetes YAML configuration as input. This gives you full control over the pod’s specifications, such as CPU, memory, image, or node selector.
+
+Here is an example of a dbt job that runs on Kubernetes using Kestra:
+
+```yaml
+id: my-dbt-job
+namespace: dev
+
+tasks:
+  - id: dbt-command
+    type: io.kestra.plugin.kubernetes.PodCreate
+
+    # Retry the task if it fails
+    retry:
+      behavior: RETRY_FAILED_TASK
+      maxAttempt: 2
+      type: constant
+      interval: PT5M
+      warningOnRetry: true
+    namespace: kestra
+
+    # Define the commands to run
+    inputFiles:
+      dbt-commands.sh: |
+        #!/bin/bash
+
+        # Exit on error
+        set -eo pipefail
+
+        # Clone the dbt example repository
+        git clone --depth 1 https://github.com/dbt-labs/jaffle_shop_duckdb.git --branch duckdb --single-branch
+
+        # Copy the dbt example repository to the working directory
+        cp -a jaffle_shop_duckdb/. .
+
+        # dbt commands to run
+        dbt deps
+        dbt build
+
+    # Define the pod specification using the Kubernetes YAML syntax
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: dbt-duckdb
+          image: ghcr.io/kestra-io/dbt-duckdb:latest
+
+          # Specify resource requirements
+          resources:
+            request:
+              cpu: "300m"
+              memory: "500Mi"
+
+          # Run the script in the container
+          command:
+            - "/bin/bash"
+            - "{{workingDir}}/dbt-commands.sh"
+
+      # Node selector to run the pod on a specific node
+      nodeSelector: {}
+```
+
+This flow will:
+- create a Kubernetes pod in the `kestra` namespace with the specified resource requirements: 300m CPU and 500Mi memory
+- clone the dbt example repository inside the pod
+- run the dbt seed and build commands
+
+![dbt-pod-create](/docs/how-to-guides/kubernetes/pod_create_dbt.png)
```

Comment:
Change link for in app doc support

Reply:
I don't know how to change it. Used same way as [here](https://github.com/aballiet/kestra-docs/blob/53d7e5b40c1ed4e67c9338856795253634e669da/content/docs/15.how-to-guides/julia.md?plain=1#L182) or [here](https://github.com/aballiet/kestra-docs/blob/53d7e5b40c1ed4e67c9338856795253634e669da/content/docs/04.workflow-components/14.concurrency.md?plain=1#L38).

Reply:
Double checked and we still use this format so should be fine